### ISSUE TITLE
Feature Request: Show number of child comments in parent comment

### ISSUE THREAD

**Issue by diego898** [etcet/HNES#141]
I think this would be a great feature

**Comment by etcet** [comment]
When you collapse a comment block it'll tell you how many child comments are below it. Do you mean it'd be nice to see the number of children before it's collapsed? I'm really hesitant to add anything if not really necessary.

**Comment by diego898** [comment]
Yes I'd like to see how many children are there before it is collapsed. In long threads, this helps me see which parents are being heavily discussed and which arent.

Also, because HN added collapsing, I now have a collapse button on the left and on the right of comments.
